index.js

```diff
@@ -476,26 +502,41 @@
   //# Boolean :: Type
   //.
   //. Type comprising `true` and `false`.
-  var Boolean_ = NullaryTypeWithUrl('Boolean', typeofEq('boolean'));
+  var Boolean_ = NullaryTypeWithUrl(Any, 'Boolean', typeofEq('boolean'));
 
   //# Date :: Type
   //.
   //. Type comprising every Date value.
-  var Date_ = NullaryTypeWithUrl('Date', typeEq('Date'));
+  var Date_ = NullaryTypeWithUrl(Any, 'Date', typeEq('Date'));
 
   //# Error :: Type
   //.
   //. Type comprising every Error value, including values of more specific
   //. constructors such as [`SyntaxError`][] and [`TypeError`][].
-  var Error_ = NullaryTypeWithUrl('Error', typeEq('Error'));
+  var Error_ = NullaryTypeWithUrl(Any, 'Error', typeEq('Error'));
+
+  //# Number :: Type
+  //.
+  //. Type comprising every primitive Number value (including `NaN`).
+  var Number_ = NullaryTypeWithUrl(Any, 'Number', typeofEq('number'));
+
+  //# ValidNumber :: Type
+  //.
+  //. Type comprising every [`Number`][] value except `NaN`.
+  var ValidNumber = NullaryTypeWithUrl(
+    Number_,
+    'sanctuary-def/ValidNumber',
+    function(x) { return !isNaN(x); }
+  );
```

Comment:
Since it's no longer possible to list the types in alphabetical order, let's decide upon some other sorting rule.